src/utils/Logger.js

```diff
@@ -1,8 +1,8 @@
-import colors from 'colors'
+import chalk from 'chalk'
 
 const defaults = {
   verbose: false,
-  silent: true
+  silent: false
```

Comment:
Is this related to the switch to `chalk`? Should be a different PR IMO so that we can discuss it separately.